.github/workflows/beta.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [nightly, beta]
+        rust: [beta]
```

Comment:
well there is no need for the matrix anymore then

Reply:
I chose to keep the matrix as it doesn't add much weight and it's easy to add more runs again.